### PR TITLE
Intly 1200

### PR DESCRIPTION
**Pull request by philbrookes**
## Motivation
A few changes required to get cronjobs working through the installation repo.

### CODE REVIEW

**Comment by pb82** [comment]
:eyes:

**Comment by pb82** [comment]
The cronjob was successfully created, but when deleting the keycloakrealm and keycloak crds i saw this error: `    time="2019-03-04T17:14:58Z" level=error msg="error syncing key (sso/minexample): failed to deprovision: failed to delete all cronjobs for sso: cronjobs.batch is forbidden: User \"system:serviceaccount:sso:default\" cannot deletecollection cronjobs.batch in the namespace \"sso\": no RBAC policy matched"`

Looks like another policy for `cronjobs/batch` is needed?

File: deploy/rbac.yaml [hunk 1]
```diff
@@ -4,6 +4,11 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: keycloak-operator
 rules:
+  - apiGroups:
+      - batch
+    resources:
+      - "cronjobs"
+    verbs: [get, list, create, update, delete]
```
Comment (pb82) [inline deploy/rbac.yaml]: `deletecollection` is missing here. Without it deprovision won't work.

Comment (philbrookes) [reply deploy/rbac.yaml]: I was trying to be too clever here, pushed a correction now.

**Comment by philbrookes** [comment]
@pb82 cheers for the review, pushed a fix now

**Pull request merged**